packages/config/src/PrismaConfig.ts

```diff
@@ -37,6 +37,36 @@ const ErrorCapturingSqlMigrationAwareDriverAdapterFactoryShape = Shape.declare(
   },
 )
 
+export type ExperimentalConfig = {
+  /**
+   * Enable experimental adapter support.
+   */
+  adapter?: boolean
+  /**
+   * Enable experimental Prisma Studio features.
+   */
+  studio?: boolean
+  /**
+   * Enable experimental external tables support.
+   */
+  externalTables?: boolean
```

Comment:
❓ Do we want to introduce this as an experimental feature at first? We wanted to GA multi schema with this coming release.

Reply:
We GA the feature, but we may find limitations with its config API in its first release. This allows us to go GA while also allowing tiny changes later, potentially